doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -188,15 +188,17 @@ described elsewhere
       .. insertprodn logical_kind logical_kind
 
       .. prodn::
-         logical_kind ::= {| @thm_token | @assumption_token }
+         logical_kind ::=
+         | {| @thm_token | @assumption_token }
+         | {| Fixpoint | CoFixpoint }
```

Comment:
don't update this stuff by hand, reset it then `make doc_gram_rsts`

Reply:
Thanks! Done (it doesn't look that good though).